constants.go

```diff
@@ -7,8 +7,9 @@ const (
 	unknownStr = "unknown"
 	ssrcStr    = "ssrc"
 
-	// Equal to UDP MTU
-	receiveMTU = 1460
+	// Raised to 1500 because this is ffmpeg's max MTU by default
+	// See pion/webrtc#1720 for full discussion
+	receiveMTU = 1500
```

Comment:
I actually thought that the MTU was supposed to be 1200 for WebRTC?

Firefox sets the max video MTU to 1200[1], and so does Chrome[2]:

1. https://searchfox.org/mozilla-central/source/dom/media/webrtc/libwebrtcglue/VideoConduit.h#49
2. https://source.chromium.org/chromium/chromium/src/+/master:third_party/webrtc/media/base/media_constants.cc;l=19?q=1200%20mtu&ss=chromium

That said, I'm not an expert on the matter. I was just relying on this fact when building a server-to-server transport for Peer Calls to prefix each RTP/RTCP packet with room name over a single port (but maybe there's a better way of doing this).

Reply:
I haven't tried this, but looks like ffmpeg has an option to specify the MTU:

> You can use "pkt_size" option when using udp as output (I don't know why it
is not the default).
> 
>     ffmpeg -i xxxx ....... udp://xx.xx.xx.xx?pkt_size=1316

Source: https://ffmpeg.org/pipermail/ffmpeg-user/2014-September/023233.html

Reply:
the mtu really should rely on the underlying network hardware... 1200 is a decent default, and should work in most places, but ideally it'd be discovered https://en.wikipedia.org/wiki/Path_MTU_Discovery

Reply:
Just FYI, 3GPP reported some typical MTU values valid for mobile networks. (usually, mobile networks have smaller MTU than wired internet connections)

> ## Annex C (informative): Link MTU considerations
>  The link MTU value that can prevent fragementation in the backbone network between the MS and GGSN/PGW depends on the actual deployement. Based on the above calculation a link MTU value of 1358 is small enough in most of the network deployements.
>  Note that using a link MTU value smaller than necessary would decrease the efficiency in the network. Moreover a UE may also apply some tunnelling (e.g., DSMIPv6 or VPN). and it is desired to use a link MTU size that assures at least 1280 octets, which is the minimum MTU size in case of IPv6, within the UE tunnel to avoid the fragmentation of the user packets within the tunnel applied in the UE.

ref: 3GPP TS 23.060 https://www.3gpp.org/ftp/Specs/archive/23_series/23.060/23060-g00.zip

Reply:
@jeremija you are right! I was helping a user in Slack, I ended up forcing the `pkt_size` [here](https://github.com/Sean-Der/goexamples/blob/webrtc/rpt-broadcast/publisher/ffmpeg.sh#L3) and it fixed everything.

This is in response to a user who was doing `ffmpeg -> Pion -> Pion`. They wanted to do everything in their LAN so MTU doesn't matter as much to them.

Before the read loop would exit without an error. Now it prints.

```
mux ERROR: 2021/03/22 08:36:11 mux: failed to read from packetio.Buffer short buffer
mux ERROR: 2021/03/22 08:36:11 mux: failed to read from packetio.Buffer short buffer
mux ERROR: 2021/03/22 08:36:11 mux: failed to read from packetio.Buffer short buffer
mux ERROR: 2021/03/22 08:36:11 mux: failed to read from packetio.Buffer short buffer
mux ERROR: 2021/03/22 08:36:11 mux: failed to read from packetio.Buffer short buffer
mux ERROR: 2021/03/22 08:36:11 mux: failed to read from packetio.Buffer short buffer
mux ERROR: 2021/03/22 08:36:11 mux: failed to read from packetio.Buffer short buffer
mux ERROR: 2021/03/22 08:36:11 mux: failed to read from packetio.Buffer short buffer
```

---

@wdouglass this isn't something we can fix with PMTU unfortunately. WebRTC doesn't support it, and in this case someone was just using the `Send` API so they are free to send w/e they want.